src/components/30-organisms/testimonials/index.react.d.ts

```diff
@@ -14,4 +14,4 @@ declare function createAXATestimonials(
   version?: string
 ): React.ComponentType<AXATestimonialsProps>;
 
-export = createAXATestimonials;
+export default createAXATestimonials;
```

Comment:
Changing the export type could be "dangerous" because it could change the way how to import that module. Please check out the differences between "Named" and "default" exports. (https://medium.com/@krishankantsinghal/named-exports-vs-default-exports-c0dffa21946d)

Reply:
Probably this is okay.
Did you test this in TS @raphael-suter ?